.github/workflows/ci-cd.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       s3mock:
         image: ghcr.io/project-zot/localstack/localstack:1.2.0
         env:
-          SERVICES: s3
+          SERVICES: s3,dynamodb
```

Comment:
`localstack` is used to mock s3. What is being used for dynamodb?

Reply:
It is used to mock dynamodb for tests